modules/lvgl/input/lvgl_common_input.c

```diff
@@ -52,3 +55,21 @@ int lvgl_input_register_driver(lv_indev_type_t indev_type, const struct device *
 
 	return 0;
 }
+
+#define LV_DEV_INIT(node_id, init_fn, ret) ret |= init_fn(DEVICE_DT_GET(node_id));
```

Comment:
`|` should be avoided for signed integers, so Fabio's early return versions seems more appropriate

Reply:
Yup, but is the checkpatch issue accetable then? `ARNING:MACRO_WITH_FLOW_CONTROL: Macros with flow control statements should be avoided`

Reply:
I think so, it's an internal macro after all...